test/stripe/country_spec_test.rb

```diff
@@ -0,0 +1,51 @@
+require File.expand_path('../../test_helper', __FILE__)
+
+module Stripe
+  class CountrySpecTest < Test::Unit::TestCase
+    should "be listable" do
+      @mock.expects(:get).once.
+      returns(make_response(country_spec_array))
+      c = Stripe::CountrySpec.list
+      
+      assert c.kind_of?(Stripe::ListObject)
+      assert c.data.kind_of? Array
+      assert_equal('/v1/country_specs', c.url)
+
+      c.data.each do |s|
+        assert s.kind_of?(Stripe::CountrySpec)
+      end
+    end
+
+    should "be retrievable" do
+      resp = make_country_spec
+      @mock.expects(:get).
+        once.
+        with('https://api.stripe.com/v1/country_specs/US', nil, nil).
+        returns(make_response(resp))
+      s = Stripe::CountrySpec.retrieve("US")
+      
+      assert_equal('/v1/country_specs/US', s.url)
+      
+      assert_equal "country_spec", s.object
+      
+      s.supported_bank_account_currencies.map{|k,v|
+        assert v.kind_of? Array
+      }
+      
+      s.supported_bank_account_currencies.kind_of?  Stripe::StripeObject
```

Comment:
I'll leave this one up to your discretion, but by testing all fields on the returned object here you're really just testing `StripeObject`'s ability to deserialize, which we have tests for elsewhere. You may want to consider just sanity checking the object's type and a couple of its fields and leave it at that.

An example:

https://github.com/stripe/stripe-ruby/blob/3a258806a26558e7c7429c1a2632acff842619ce/test/stripe/account_test.rb#L22,L34